metadata-db-app/src/migrations/048_setup_variables_table.clj

```diff
@@ -0,0 +1,59 @@
+(ns migrations.048-setup-variables-table
+  (:require [clojure.java.jdbc :as j]
```

Comment:
Same comment about updating the `require` indents here to use our adopted style.